generator-lateralus/app/templates/app/scripts/main.js

```diff
@@ -2,36 +2,36 @@
 'use strict';
 
 require.config({
-  baseUrl: '/'
-  ,map: {
+  baseUrl: '/',
+  map: {
     '*': {
       underscore: 'lodash'
     }
-  }
-  ,shim: {
+  },
+  shim: {
     bootstrap: {
-      deps: ['jquery']
-      ,exports: 'jquery'
+      deps: ['jquery'],
+      exports: 'jquery'
     }
-  }
-  ,paths: {
-    text: 'bower_components/requirejs-text/text'
-    ,jquery: 'bower_components/jquery/dist/jquery'
-    ,backbone: 'bower_components/backbone/backbone'
-    ,lodash: 'bower_components/lodash/lodash'
-    ,mustache: 'bower_components/mustache/mustache'
-  }
-  ,packages: [{
-    name: 'lateralus'
-    ,location: 'bower_components/lateralus/scripts'
-    ,main: 'lateralus'
+  },
+  paths: {
+    text: 'bower_components/requirejs-text/text',
+    jquery: 'bower_components/jquery/dist/jquery',
+    backbone: 'bower_components/backbone/backbone',
+    lodash: 'bower_components/lodash/lodash',
+    mustache: 'bower_components/mustache/mustache'
+  },
+  packages: [{
+    name: 'lateralus',
+    location: 'bower_components/lateralus/scripts',
```

Comment:
Thinking about this a bit more, I think the `generator` part of this project is creating more problems than its solving, at this point.  That is, I don't think anybody is using it, and its existence is creating a maintenance burden.

I'd like to just `rm -rf generator-lateralus` before the 1.0.0 release, which I would do in a followup PR.  What do you think about that, rather than making additional changes to it in this PR?